src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/IncludeRewriter.cs

```diff
@@ -27,19 +28,17 @@ public override Expression VisitSqlRoot(SqlRootExpression expression, object con
                 return expression;
             }
 
-            bool containsInclude = false;
+            bool containsInclude = expression.TableExpressions.AsEnumerable().Where(e => e.SearchParameterQueryGenerator is IncludeQueryGenerator).Any();
```

Comment:
What is the scenario when `Kind == Include` and  `!(SearchParameterQueryGenerator is IncludeQueryGenerator)`

Reply:
TBH I don't know that there is. I was consistent with the code that existed before. 
I'll try to use Kind and make sure all tests pass